src/editor/pages/api/flows/create.ts

```diff
@@ -0,0 +1,34 @@
+import { v4 as uuidv4 } from 'uuid'
+
+const handler = async ( req: any, res: any ): Promise<void> => {
+  const date = new Date().toISOString()
+  const id = uuidv4()
+
+  const newFlow = {
+    id: id,
+    name: 'DataFlow-' + id.toString().substring( 0, 8 ),
+    status: 'Active',
+    lastEdited: date,
+    definition: ''
+  }
+
+  const url = process.env.API_BASE_URL + '/api/DataFlow'
```

Comment:
use the concatination with back ticks => `${process.env.API_BASE_URL}/api/DataFlow`